security/totp-cli/Portfile

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/yitsushi/totp-cli v1.1.13
```

Comment:
The format for `go.setup` is `{module} {version} {version_prefix}`

`go2port` has a bug where it doesn't do the right thing around separating the prefix from the version number when generating a Portfile.

So can you change this to:
`go.setup            github.com/yitsushi/totp-cli 1.1.13 v`

Also can you add a `revision` field? Revision is present on ports so that it can be bumped when changes are being made where the main version isn't changing.

```
go.setup            github.com/yitsushi/totp-cli 1.1.13 v
revision            0
```

Reply:
Also please ensure any changes you make are just revisions of the one commit, as per the [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)

Reply:
Thanks @herbygillot ! I've made changes you requested.